src/main/webapp/app/shared/sort/sort.directive.ts

```diff
@@ -0,0 +1,39 @@
+import { Directive, EventEmitter, Input, Output } from '@angular/core';
+
+@Directive({
+    selector: '[jhiSort]',
+})
+export class SortDirective<T> {
```

Comment:
I find this order very confusing. Also that the getter is an input. Can we put the methods under the attributes and move the input away from the getter? Or it this standard behaviour with directives?